src/plugins/intel_gpu/src/graph/impls/ocl/primitive_base.hpp

```diff
@@ -276,6 +276,19 @@ struct typed_primitive_impl_ocl : public typed_primitive_impl<PType> {
                 (std::accumulate(gws.begin(), gws.end(), 1, std::multiplies<size_t>()) == 0);
         }
     }
+
+    void set_kernels(std::map<const std::string, kernel::ptr>& kernels) {
```

Comment:
Please, fix compilation error https://dev.azure.com/openvinoci/dldt/_build/results?buildId=500845&view=logs&j=e0e676f4-9664-538a-6d6b-aa53516aa09f&t=839cb6ab-4ea8-51d4-e102-9a106ad673bf&l=4373
Compiler is clang

Reply:
https://github.com/openvinotoolkit/openvino/pull/15948
Created fix.